### PR TITLE
US1.1: add a new booking

### PR DESCRIPTION
**Pull request by soufcod3**

### CODE REVIEW

**Review by mcbeloy — APPROVED** [review]

**Review by mcbeloy — APPROVED** [review]

**Review by FouziaAs — DISMISSED** [review]

**Comment by FouziaAs** [comment]
ok

File: src/Controller/GeneralInfoController.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        $booking = new Booking();
+
+        $form = $this->createForm(GeneralInfoType::class, $booking);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $entityManager = $this->getDoctrine()->getManager();
+            $entityManager->persist($booking);
+            $entityManager->flush();
+            return $this->redirectToRoute('booking_index');
```
Comment (RomainClair) [inline src/Controller/GeneralInfoController.php:32]: The booking_index route does not exists

**Review by RomainClair — CHANGES_REQUESTED** [review]
As the GeneralInfoType is used to fill a Booking entity, you should have Constraints in Booking to validate your input.
You could use IntegerType instead of TextType for numeric values

There's no validation either for Client (filled by the ClientType) and Agent (filled via AgentType)

**Review by iStyyx — APPROVED** [review]
Looks better

**Review by RomainClair — CHANGES_REQUESTED** [review]
The migration file is incoherent with the database as it is in the dev branch

**Review by mcbeloy — APPROVED** [review]

**Comment by mcbeloy** [comment]
It's good

**Review by mcbeloy — APPROVED** [review]

**Review by RomainClair — APPROVED** [review]

**Pull request merged**